### DISCUSSION TITLE
Programming language and UI framework?

### DISCUSSION THREAD

**Discussion by luximus-hunter** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#207] [Q&A]
Hey hoi,

First of all: wonderful piece of software. Absolutely love it. ♥️

Second: what programming language does this project use? As far as I can see it's JavaScript running in electron. Is that correct?

Also, what UI framework do you use? If any at all of course.

Thanks for the response in advance. ☺️

**Comment by Melvin-Abraham** [comment]
@luximus-hunter Sorry for the late reply.

> what programming language does this project use? As far as I can see it's JavaScript running in electron. Is that correct?

Yes you are absolutely correct!

> what UI framework do you use?

Currently, it does not use any framework. It's made in vanilla (pure) JavaScript. Though this might change soon in future...

**Reply by luximus-hunter** [reply]
Cool! I might look at contributing in the near future. I'm studying software and web development, and I specially like JavaScript and ReactJS. I will see what I can do when i have free time.
